software/open-quartermaster.yml

```
@@ -0,0 +1,11 @@
+name: "Open QuarterMaster"
+website_url: "https://openquartermaster.com/"
+source_code_url: "https://github.com/Epic-Breakfast-Productions/OpenQuarterMaster"
+description: "Powerful inventory management system, designed to be flexible and scalable."
+licenses:
+  - GPL-3.0
+platforms:
+  - Deb
```

```suggestion
  - deb
```

platforms are case-sensitive